integration_test/oracle_module/set_feeder_test.yaml

```diff
@@ -0,0 +1,28 @@
+- name: Test setting a different feeder account
+  inputs:
+    # Get admin addr
+    - cmd: printf "12345678\n" | seid keys list --output json | jq ".[] | select (.name==\"node_admin\")" | jq -r .address
+      env: ADMIN_ADDR
+    # Get admin val addr
+    - cmd: printf "12345678\n" | seid keys show node_admin --bech val --output json | jq -r ".address"
+      env: ADMIN_VAL_ADDR
+    # create new feeder addr
+    - cmd: printf "12345678\ny\n" | seid keys add integration_test_feeder --output json | jq -r ".address"
+      env: NEW_FEEDER_ADDR
+    # get old feeder addr
+    - cmd: seid q oracle feeder $ADMIN_VAL_ADDR --output json | jq -r ".feeder_addr"
+      env: OLD_FEEDER_ADDR
+    # send bank funds to new feeder to populate account
+    - cmd: printf "12345678\n" | seid tx bank send admin $NEW_FEEDER_ADDR 1sei -b block -y --fees 2000usei --output json | jq -r ".code"
+    # set feeder
+    - cmd: printf "12345678\n" | seid tx oracle set-feeder $NEW_FEEDER_ADDR -b block -y --from node_admin --fees 2000usei --output json | jq -r ".code"
+      env: SET_FEEDER_ERROR_CODE
+    # do a vote and ensure code 0
+    - cmd: printf "12345678\n" | seid tx oracle aggregate-vote 1.5ueth $ADMIN_VAL_ADDR -b block -y --from $NEW_FEEDER_ADDR --gas 0 --output json | jq -r ".code"
+      env: NEW_FEEDER_VOTE_ERROR_CODE
+    # set old feeder addr again
+    - cmd: printf "12345678\n" | seid tx oracle set-feeder $OLD_FEEDER_ADDR -b block -y --from node_admin --fees 2000usei --output json | jq -r ".code"
```

Comment:
Can we add verification for the error count rate too? It was broken on master branch for a couple of weeks. we caught it as a partner reported it

Reply:
wdym it was broken? what was broken about it? do you have additional context into this? if youre referring to vote penalties its a little hard to test exactly on those since those are updated every block (vote window) and are easier to cover the test cases via unit testing for oracle mid/end block logic.

Reply:
yeah +1 to verify the signing success

Reply:
We can just use seid to query the success and make sure it's above a value after certain blocks or amount of time

Reply:
for that, I can add a separate integration test that just asserts those values since the feeders are already running, would that be ok?

cc @BrandonWeng for more context into the error case we previosuly saw?

Reply:
btw, I think we can add a TODO to switch docker to use the sei-chain example price feeder config, instead of the one that we copied over to docker folder